### ISSUE TITLE
[Maps] Correct the tab order of the MapLibre link and info

### ISSUE THREAD

**Issue by tallulahkay** [civiform/civiform#11448]
Currently, the info icon gets tabbed to first, and then the link. The order should be switched.

**Comment by tallulahkay** [comment]
Since it's a disclosure widget, the tab order is correct

https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/
